packages/sitecore-jss-nextjs/src/components/NextImage.tsx

```diff
@@ -0,0 +1,164 @@
+import { mediaApi } from '@sitecore-jss/sitecore-jss/media';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { getEEMarkup } from '@sitecore-jss/sitecore-jss-react';
+import Image, { ImageLoader, ImageLoaderProps, ImageProps as NextImageProps } from 'next/image';
+export interface ImageFieldValue {
```

Comment:
You have a lot of duplicate code in this component, many interfaces you took from the `Image` component, so you should have shared interfaces instead of duplicates. Please, compare these components and find the same things